helm/nessie/templates/service.yaml

```diff
@@ -39,6 +39,14 @@ spec:
       protocol: TCP
     {{- end }}
   sessionAffinity: {{ .Values.service.sessionAffinity }}
+  {{- if .Values.service.clusterIP }}
+  clusterIP: {{ .Values.service.clusterIP }}
+  {{- end }}
+  externalTrafficPolicy: {{ .Values.service.externalTrafficPolicy }}
```

Comment:
I think we still have a problem here: the tests are failing because this value can only be set if the service is of type `LoadBalancer`.

So, I think we need to wrap this line in an if/else template block.

Reply:
That's a good catch. It will also work for `NodePort` as well.

Would something like this work?

```
{{- if and $svc.externalTrafficPolicy (or (eq $svc.type "LoadBalancer") (eq $svc.type "NodePort")) }}
```

Or would you prefer this separated out with multiple if-statements wrapping each other? Not sure what would go in the _else_ block.

Reply:
I think your suggestion is fine 👍